crates/standalone/src/control_db.rs

```diff
@@ -83,7 +83,9 @@ impl ControlDb {
 
     pub fn spacetime_reverse_dns(&self, database_identity: &Identity) -> Result<Vec<DomainName>> {
         let tree = self.db.open_tree("reverse_dns")?;
-        let value = tree.get(database_identity.to_byte_array())?;
+        // TODO: everywhere in this file should use `to_be_byte_array`.
+        // We may want to implement some sort of trait to ensure that it is used consistently.
+        let value = tree.get(database_identity.to_be_byte_array())?;
```

Comment:
Do we need to change this? Maybe just leave it for consistency with the existing code? Or does it differ from how private does it?

Reply:
I *believe* this is how it worked before https://github.com/clockworklabs/SpacetimeDB/pull/1616 was merged, and #1616 accidentally transposed it. Let me double check in `Private`.

Reply:
🙏

Reply:
In the control module in `Private`, everything is just stored in a `SpacetimeDB` table.

I did this here just because, when you're inspecting a `standalone` sled database, it seems like it would be more intuitive for the byte arrays indexing things to match the printed forms of addresses/identities. But it isn't a huge deal.

Reply:
Can you just leave them little endian so that this isn't a breaking change from 0.12?

Reply:
Sure.